src/main/play-doc/operation/Install.md

```diff
@@ -830,4 +1372,4 @@ NAME           HOST       USER  STATE  ID
 visualizer-1   10.0.3.75  root    R    6e68f055d1f5715ad3ff19172fa5efaf_0f6e119c-9288-425a-89e3-36379dcaccda
 ```
 
```

Comment:
Should we curl an endpoint from `conduct services` or the likes?
IOW, as an operator, where's my dmarc interface? The service might be returning 200's, but if it's not reachable, it's not verified.